validation/forms.py

```diff
@@ -130,15 +130,17 @@ class Meta:
 
 class EditIssueWithPhrase(forms.ModelForm):
     transcription = forms.CharField(
-        max_length=200,
+        max_length=412,
```

Comment:
In other django projects, I’ve started changing these random length values to reference a `DEFAULT_MAX_LENGTH=500` constant at the top of the file.